src/software/thunderscope/robot_communication.py

```diff
@@ -81,7 +81,7 @@ def __send_estop_state(self):
                 self.current_proto_unix_io.send_proto(
                     EstopState, EstopState(is_playing=self.estop_reader.isEstopPlay())
                 )
-            time.sleep(0.1)
+                time.sleep(0.1)
```

Comment:
Wow. This improved the performance of diagnostics by ~3000% on my computer. It loads instantly now :sweat_smile:

Reply:
Us yesterday:
![image](https://user-images.githubusercontent.com/28585597/226187234-ed730ecc-ba52-40a3-8b9d-df280f410ed8.png)